apps/financial-aid/web-veita/src/components/Nav/Nav.tsx

```diff
@@ -88,6 +88,38 @@ const Nav = ({ showInMobile }: Props) => {
 
       <Box display="block" marginBottom={2} marginTop={4}>
         <Box marginBottom={2}>
+          <button
+            className={cn({
+              [`${sideNavButtonStyles.sideNavBarButton} navBarButtonHover`]: true,
+              [`${sideNavButtonStyles.activeNavButton}`]:
+                router.pathname === Routes.settings.search,
+            })}
+            onClick={() => router.push(Routes.settings.search)}
+          >
+            <Icon
+              icon="search"
+              type="outline"
+              color="blue400"
+              className={sideNavButtonStyles.sideNavBarButtonIcon}
+            />
+            <Text> Leit</Text>
+          </button>
```

Comment:
EmployeeSideNavItems has the filters of the applications, and this belongs to the footer of the nav so it would be hard to place it there. However maybe change the name applicationfilters and then have this button in component EmployeeSideNavItems?

Reply:
We should check this out together 😄